File: Dtos/HealthFacilities/UpdateHealthFacilityDto.cs
namespace OskApi.Dtos.HealthFacilities
{
    public class UpdateHealthFacilityDto
    {
        public Guid Id { get; set; }
        public Guid HealthFacilityTypeId { get; set; }
        public string Name { get; set; } = null!;
        public string? Address { get; set; }
        public string? PhoneNumber { get; set; }
        public string? Email { get; set; }
        public string? TaxNumber { get; set; }
        public string? CorporationName { get; set; }
    }
}

File: OskUI2/OskUI2_temp/src/app/dtos/healthFacility/update-health-facility.dto.ts
export class UpdateHealthFacilityDto {
  id: string = '';
  healthFacilityTypeId: string = '';
  typeName: string = ''; // sadece görüntüleme için
  name: string = '';
  address?: string;
  phoneNumber?: string;
  email?: string;
  taxNumber?: string;
  corporationName?: string;
}
